### PR TITLE
Improved the display new icons

### PR DESCRIPTION
**Pull request by areksolek**
Improved the display new icons

### CODE REVIEW

**Comment by mariuszkrzaczkowski** [comment]
## Migration Recommended

It appears you are still using the CoPilot OAuth App - we recommend you [migrate to the GitHub App](https://github.com/apps/blackduck-copilot/installations/new/permissions?suggested_target_id=10201412&repository_ids[]=28060246)

**Comment by mariuszkrzaczkowski** [comment]
## Black Duck Security Report

Merging #11289 into developer will not change security risk.

[Click here to see full report](https://copilot.blackducksoftware.com/github/repos/YetiForceCompany/YetiForceCRM/pull-requests/11289)

File: modules/Vtiger/models/Menu.php [hunk 1]
```diff
@@ -191,27 +192,28 @@ public static function getModuleNameFromUrl($url)
...
 				return '<img src="' . $iconName . '" alt="' . $title . '" title="' . $title . '" class="c-menu__item__icon" />';
 			}
 		}
 		if (!empty($menu['icon'])) {
-			if (strpos($menu['icon'], 'fa-') !== false) {
+			if (false !== strpos($menu['icon'], 'fa-')) {
 				return '<span class="fa-lg fa-fw ' . $menu['icon'] . ' c-menu__item__icon"></span>';
-			} elseif (strpos($menu['icon'], 'adminIcon-') !== false || strpos($menu['icon'], 'userIcon-') !== false || strpos($menu['icon'], 'AdditionalIcon-') !== false) {
+			}
+			if (false !== strpos($menu['icon'], 'adminIcon-') || false !== strpos($menu['icon'], 'userIcon-') || false !== strpos($menu['icon'], 'AdditionalIcon-') || false !== strpos($menu['icon'], 'yfi-')) {
```
Comment (rskrzypczak) [inline modules/Vtiger/models/Menu.php]: yfm

File: modules/Vtiger/models/Menu.php [hunk 2]
```diff
@@ -126,21 +127,21 @@ public static function getBreadcrumbs($pageTitle = false)
...
 						$breadcrumbs[] = $title;
 					}
 				} else {
 					if ($pageTitle) {
 						$breadcrumbs[] = ['name' => App\Language::translate($pageTitle, $qualifiedModuleName)];
-					} elseif ($view === 'Edit' && $request->getRaw('record') === '' && $request->getRaw('parent_roleid') === '') {
+					} elseif ('Edit' === $view && '' === $request->getRaw('record') && '' === $request->getRaw('parent_roleid')) {
 						$breadcrumbs[] = ['name' => App\Language::translate('LBL_VIEW_CREATE', $qualifiedModuleName)];
-					} elseif ($view != '' && $view != 'List') {
+					} elseif ('' !== $view && 'List' !== $view) {
```
Comment (rskrzypczak) [inline modules/Vtiger/models/Menu.php]: ?

**Pull request merged**